src/main/resources/application.properties

```diff
@@ -42,6 +42,7 @@ invalid.username.size=Your login must be between 2 and 50 characters, do not use
 empty.username.field=Login field is required
 invalid.email=Please enter valid email address (Ex: username@example.com)
 invalid.username=Please enter valid login, do not use space
+invalid.token=You must to authenticated
```

Comment:
Change it to "You must be authenticated".

Reply:
Done in commit "caaf656"